src/MongoDB.Analyzer/Core/Utilities/StringExtensions.cs

```diff
@@ -21,4 +21,7 @@ internal static class StringExtensions
 
     public static bool IsNotEmpty(this string str) =>
         !string.IsNullOrWhiteSpace(str);
+
+    public static bool Contains(this string source, string target, StringComparison comp) =>
+        source != null && target != null && source.IndexOf(target, comp) >= 0;
```

Comment:
I did not realize `string.Contains` was not available in .net472

Reply:
Yea the only alternative was IndexOf using the StringComparison values